vllm/engine/arg_utils.py

```diff
@@ -648,16 +649,38 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
         parallel_group.add_argument(
             "--enable-expert-parallel",
             **parallel_kwargs["enable_expert_parallel"])
+        parallel_group.add_argument("--eplb-config",
+                                    **parallel_kwargs["eplb_config"])
         parallel_group.add_argument("--enable-eplb",
                                     **parallel_kwargs["enable_eplb"])
-        parallel_group.add_argument("--num-redundant-experts",
-                                    **parallel_kwargs["num_redundant_experts"])
-        parallel_group.add_argument("--eplb-window-size",
-                                    **parallel_kwargs["eplb_window_size"])
-        parallel_group.add_argument("--eplb-step-interval",
-                                    **parallel_kwargs["eplb_step_interval"])
-        parallel_group.add_argument("--eplb-log-balancedness",
-                                    **parallel_kwargs["eplb_log_balancedness"])
+        parallel_group.add_argument(
+            "--num-redundant-experts",
+            type=int,
+            help=
+            '[DEPRECATED] --num-redundant-experts will be removed in v0.12.0.',
+            default=EPLBConfig.num_redundant_experts,
+            deprecated=True)
```

Comment:
I don't think these deprecated CLI args are used anywhere, could they be used to override `self.eplb_conifg` near line ~1200 if they are non-default?

Reply:
It in this code to set `eplb_config.num_redundant_experts` field value. we dno't override self.eplb_conifg near line ~1200, I think the current code is good, because when vllm version to 0.12.0 after, only remoe about `deprecated` code, No other code needs to be modified.
```
@num_redundant_experts.setter
    @deprecated(
        "`num_redundant_experts` is deprecated and has been replaced with "
        "`eplb_config.num_redundant_experts`. This will be removed in v0.12.0. "
        "Please use `eplb_config.num_redundant_experts` instead.")
    def num_redundant_experts(self, value: int):
        self.eplb_config.num_redundant_experts = value
```

Reply:
During the deprecation period, the old CLI args should still work. Right now the deprecated CLI args do nothing, which will be unexpected for users.